docs/integration-suse-ai.adoc

```diff
@@ -0,0 +1,221 @@
+// Copyright 2025 SUSE LLC
+// SPDX-License-Identifier: Apache-2.0
+
+== SUSE AI Integration
+
+This document provides a guide on how to use the `Trento MCP Server` with SUSE AI, including configuration and deployment instructions.
+
+=== Getting Started with SUSE AI
+
+SUSE AI is a platform that allows you to deploy and manage AI models and applications in a Kubernetes environment. It provides tools for model management, deployment, and integration with various AI frameworks.
+
+Refer to the official https://documentation.suse.com/suse-ai/1.0/[SUSE AI documentation] for detailed information. In this guide, we will focus on deploying the `Trento MCP Server` using SUSE AI, specifically with Ollama and Open Web UI. Always refer to the latest instructions in the https://documentation.suse.com/suse-ai/1.0/html/AI-deployment-intro/index.html[SUSE AI deployment guide] for the most accurate and up-to-date information.
+
+=== Prerequisites
+
+This guide assumes:
+
+* you have a Kubernetes cluster set up and running (with an ingress controller and cert-manager installed)
+* you have access to the SUSE Application Collection.
+* if you want to run Ollama models, you have either:
```

Comment:
Remove `if you want`  better --> `To run Ollama models, you have either:

Check out the style guide from the docs team :) https://documentation.suse.com/style/current/html/style-guide-adoc/sec-language.html#sec-tone

Reply:
Good point! I just wrote it quickly and, TBH, I did proofread it against our style guide.

That said, I think we should consider (maybe for the docs site?) using a linter. In the past, I've worked with [Vale](https://vale.sh/docs/install) and it catches several gotchas.